webviews/components/user.tsx

```diff
@@ -8,13 +8,13 @@ import { IAccount, IActor, ITeam, reviewerLabel } from '../../src/github/interfa
 import { Icon } from './icon';
 
 export const Avatar = ({ for: author }: { for: Partial<IAccount> }) => (
-	<a className="avatar-link" href={author.url} title={author.url}>
+	<>
```

Comment:
Since this `Avatar` is used in other places outside the create view we'll need to keep the `a` in those cases. I'll push a change to add an argument for this.